client/response.go

```diff
@@ -77,11 +77,26 @@ func (c *Client) Guilds() ([]Channel, error) {
 
 func (c *Client) GuildMessages(channel *Channel, me *Me, seek *int) (*Messages, error) {
 	endpoint := fmt.Sprintf(endpoints["guild_msgs"], channel.ID, me.ID, *seek, messageLimit)
+	endpoint_ := fmt.Sprintf(endpoints["guild_channels"], channel.ID)
+
+	var channels []Me
```

Comment:
This type should be `[]Channel`.

Reply:
`[]Channel` is for DM channels, the struct has the `Recepients` aswell
We just need to filter out IDs so i used `[]Me` cuz it only has an ID field